### PR TITLE
First round of renamings

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

File: cmake/export.cmake [hunk 1]
```diff
@@ -12,50 +12,50 @@
...
+    ${CMAKE_CURRENT_BINARY_DIR}/cpp_bindgen-config-version.cmake
     DESTINATION lib/cmake
     )
     
-set(__C_BINDINGS_CMAKE_DIR "\${gt_c_bindings_MODULE_PATH}")  #TODO refactor the variable names gt_c_bindings_MODULE_PATH, etc.
-set(__C_BINDINGS_SOURCE_DIR "\${gt_c_bindings_SOURCES_PATH}")
-set(__C_BINDINGS_INCLUDE_DIR "\${gt_c_bindings_INCLUDE_PATH}")
-configure_file(cmake/gt_bindings.cmake.in
-    ${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/build-install/lib/cmake/gt_bindings.cmake
+set(__C_BINDINGS_CMAKE_DIR "\${cpp_bindgen_MODULE_PATH}")  #TODO refactor the variable names cpp_bindgen_MODULE_PATH, etc.
```
Comment (lukasm91) [inline cmake/export.cmake:37]:
```suggestion
set(__C_BINDINGS_CMAKE_DIR "\${cpp_bindgen_MODULE_PATH}")
```
I think that's what this PR is intended for, right?

**Review by lukasm91 — APPROVED** [review]
I didn't try out exporting yet; just looked at the code. Will do this experiment in the end.

File: cmake/export.cmake (hunk 1 above)
Comment (havogt) [reply cmake/export.cmake:37]: No this TODO is for syncing the names. Sometimes we call it MODULE_PATH, sometimes CMAKE_DIR, etc.

**Pull request merged**